src/scancode/api.py

```diff
@@ -200,7 +208,9 @@ def get_licenses(location, min_score=0,
     detected_spdx_expressions = []
     return dict([
         ('licenses', detected_licenses),
+        ('unknown_licenses', unknown_licenses),
```

Comment:
I would rather have licenses, license_expressions, unknown_licenses, unknown_license_expressions in this order everywhere.... this make more sense to group unknowns together.